.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
```

Comment:
what's the purpose of this file?

Reply:
@systembugtj houndci-bot did not resolve esversion: 6 put into .jshintrc file in https://github.com/systembugtj/ember-cli-panzoom/pull/3
I might be overcomplicating - will remove it! Thanks!